_component_design/breadcrumbs.md

```diff
@@ -18,30 +18,50 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/breadcrumbs/
 
 <section class="static-section" markdown="1">
 
-Use breadcrumbs on Rackspace sub-pages to let the user know where they are in the page hierarchy. Do not use breadcrumbs on the first layer of any multi-layered module, only use breadcrumbs on the second level of nesting and deeper within a module.
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use breadcrumbs when you want to improve web application way-finding and navigation. Breadcrumbs help the user understand their location within an application, and provide an easy way for the user to navigate out of their current view.
+
+{% endcolumn %}
+
+</div>
 
 </section>
 
 <section class="static-section" markdown="1">
 
-## Introduction
-
 <div class="hxRow"  markdown="1">
+
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-### When to use
+## Best practices
 
-When using breadcrumbs, keep the following in mind:
+Consider the following best practices when you construct breadcrumbs:
 
-- Keep the breadcrumbs brief and as minimal as possible.
-- Ensure the breadcrumbs go on the top of the page, above the header, and only show up on the second level of your app and deeper.
+- Label the breadcrumb item to match the page title to which it navigates.
+- If the page title is too long, use a shortened variation of the page title. For example, instead of `Billing overview`, use `Overview`. Shorter labels conserve horizontal space.
+- Place the breadcrumbs at the top of the page, above the header.
+- Only use breadcrumbs on the second level and deeper of a multi-layered application. Do use breadcrumbs on the first layer of a multi-layered application.
+
```

Comment:
always show the top hierarchical level as the first bread crumb. 
Always show current location as the last crumb.
Always show the  one level above current location for the middle crumb.

___
Top > [...] > 2 levels above Current > Level above Current > Current 

Need a visual for this^

Reply:
I'm not entirely sure what you mean by your most recent comment. I came up with "Although breadcrumbs do not appear on the top-level page, ensure that the spacing in the header is consistent with child pages where breadcrumbs do appear."

Am I understanding your intent correctly?

Reply:
@chri2547 is this discussion resolved?

Reply:
Yes, this is resolved. Done.